ui/components/app/detected-token/detected-token-selection-popover/detected-token-selection-popover.js

```diff
@@ -28,9 +34,20 @@ const DetectedTokenSelectionPopover = ({
     selectedTokens.length === detectedTokens.length
       ? `All`
       : `(${selectedTokens.length})`;
+  const detectedTokensDetails = detectedTokens.map(
```

Comment:
NIT: we can move this const into the `onClose` function since it's not used outside of the function. This will also prevent `detectedTokens.map` from being called until it's requested